client/packages/common/src/intl/utils/IntlUtils.ts

```diff
@@ -125,6 +126,15 @@ export const useIntlUtils = () => {
     return t(localeKey, Formatter.fromCamelCase(serverKey));
   };
 
+  const translateReportName = (code: string, name: string) => {
+    const localeKey = `report.${code}`
+    return isLocaleKey(localeKey) ? t(localeKey) : name;
+  }
```

Comment:
I wanted to make this generic for any string / prefix combination, but thought this would make it a bit more readable where used to make it specific to reports.